assets/pages/digital-subscription-landing/digitalSubscriptionLanding.scss

```diff
@@ -25,7 +25,9 @@
 #digital-subscription-landing-page-uk,
 #digital-subscription-landing-page-us,
 #digital-subscription-landing-page-au,
-#digital-subscription-landing-page-eu {
+#digital-subscription-landing-page-eu,
+#digital-subscription-landing-page-nz,
+#digital-subscription-landing-page-ca {
```

Comment:
not for this PR but do we need all these ids? what is the purpose?

Reply:
I don't think they really serve a purpose, the idea was to allow per country customisation but since we don't really do that I think we should remove them.

Reply:
Gonna card this in health